ingestion/Dockerfile.ci

```diff
@@ -82,7 +82,7 @@ WORKDIR /home/airflow/openmetadata-airflow-apis
 RUN pip install "."
 
 WORKDIR /home/airflow/ingestion
-ARG INGESTION_DEPENDENCY="all"
+ARG INGESTION_DEPENDENCY
```

Comment:
@ayush-shah Why do we want to remove the default value of argument here ?

Reply:
so if i modify env `INGESTION_DEPENDENCY` in the docker-compose file under ingestion container service:
it doesn't take that into account

Reply:
This is the argument / environment variable you need to pass during Docker build phase and not during the running the docker compose!

Reply:
I think that param is working as expected there. We should close this